modules/persistent-storage-csi-manila-install-driver.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
+
+[id="persistent-storage-csi-manila-storage-class_{context}"]
+= Installing the OpenStack Manila CSI Driver Operator
+
+The CSI Manila Operator is not installed in {product-title} by default. Use the following procedure to install and configure this Operator to enable the CSI Manila driver in your cluster.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* NFS share types are used for provisioning.
+//This prereq won't be required unless there are custom install instructions that are added to OCP docs:
+//* The OpenStack Manila service has been correctly set up and is accessible from the {product-title} cluster.
+
+.Procedure
+
+To create the `manila-csi` project from the web console, follow these steps:
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Projects*.
+
+. Click *Create Project*.
+
+. To create the installed namespace, enter *openshift-csi-driver-manila-operator* in the *Name* field.
+
+. Edit the fields for the *Display Name* and *Description* using the values that you want.
+
+. Click *Create*.
```

Comment:
IMO, project / namespace creation is not necessary. OLM will create the namespace automatically when user follows this:

> On the *Install Operator* page, be sure that *A specific namespace on the cluster* is selected. Select *openshift-csi-driver-manila-operator* from the drop-down menu.

Reply:
Makes sense. I wasn't 100% sure about that as I couldn't test in 4.5 yet. I've removed the steps to create project/namespace.